src/test/java/tests/MilestoneAliasTests.java

```diff
@@ -160,6 +161,117 @@ public void replaceMilestoneAliasesTest() {
     }
 
     @Test
+    public void milestoneAlias_onlyOpenMilestoneWithoutDueDate_openMilestoneConsideredAsCurrent() {
+        //ongoing milestone -> open && (!overdue || (overdue && openIssues > 0))
+
+        TurboMilestone msFinished0 = new TurboMilestone(REPO, 0, "finished0");
+        msFinished0.setDueDate(Optional.of(LocalDate.now()));
+        msFinished0.setOpen(false);
+
+        TurboMilestone msFinished1 = new TurboMilestone(REPO, 1, "finished1");
+        msFinished1.setDueDate(Optional.of(LocalDate.now().minusDays(1)));
+        msFinished1.setOpen(false);
+
+        TurboMilestone msFinished2 = new TurboMilestone(REPO, 2, "finished2");
+        msFinished2.setDueDate(Optional.of(LocalDate.now().minusDays(2)));
+        msFinished2.setOpen(false);
+
+        TurboMilestone msOngoing0 = new TurboMilestone(REPO, 0, "ongoing0");
+        msOngoing0.setDueDate(Optional.empty());
+        msOngoing0.setOpen(true);
+
+        TurboMilestone msOngoing1 = new TurboMilestone(REPO, 3, "ongoing1");
+        msOngoing1.setDueDate(Optional.empty());
+        msOngoing1.setOpen(true);
+
+        TurboMilestone msOngoing2 = new TurboMilestone(REPO, 4, "ongoing2");
+        msOngoing2.setDueDate(Optional.empty());
+        msOngoing2.setOpen(true);
+
+        // only one milestone, open and no due date
+        model = TestUtils.singletonModel(new Model(REPO,
+               new ArrayList<>(),
+                new ArrayList<>(),
+                new ArrayList<>(Arrays.asList(msOngoing0)),
+                new ArrayList<>()
+        ));
+
+        FilterExpression expr = Qualifier.replaceMilestoneAliases(model, Parser.parse("milestone:current"));
+        List<Qualifier> milestoneQualifiers = expr.find(Qualifier::isMilestoneQualifier);
+        milestoneQualifiers.stream().forEach(msQ -> {
+            assertEquals("ongoing0", msQ.getContent().get());
+        });
+
+        // 2 finished milestones with due dates, and 1 open milestone with no due date
+        // the finished milestone should be able to be referred with current-1 and current-2
+        model = TestUtils.singletonModel(new Model(REPO,
+                new ArrayList<>(),
+                new ArrayList<>(),
+                new ArrayList<>(Arrays.asList(msFinished1, msFinished2, msOngoing1)),
+                new ArrayList<>()
+        ));
```

Comment:
This code snippet can be extracted into an `Object mother` method (i.e. a utility method that creates objects)? e.g. `createModel(TurboMilestone... milestones)`